app/src/main/java/com/tma/sparking/services/http/GsonParser.java

```diff
@@ -21,7 +21,7 @@
 /**
  * Custom json parser for parsing an awkward json data...
  */
-public class GsonParser {
+class GsonParser {
```

Comment:
GsonParser is not an descriptive name. Please rename it ex: FieldServiceGsonParser